### ISSUE TITLE
Monte Carlo Integration

### ISSUE THREAD

**Issue by insertinterestingnamehere** [byuimpact/numerical_computing#531]
The current lab on Monte Carlo Integration needs some added material and some clarification in the problems. It would be good to have them put something together where they can integrate a callable function over a very high number of dimensions. Currently the problems are vague. We also need to give an explanation that gives the students more intuition as to how this works. Finding the area of the unit sphere would be a really nice problem. Showing how that relates to the method introduced in the lab would also be really nice. Also, if we want to compare to quadrature, we should probably give them a function that integrates a callable function recursively over an arbitrary number of dimensions using the built in quadrature function instead of doing Riemann sums.

**Comment by rachelwebb** [comment]
I think a lot of these issues were addressed before I got the lab. What do you think of the current state of the lab?

We could add a problem to find the area of the unit sphere in higher dimensions.

**Comment by insertinterestingnamehere** [comment]
Hmm, yep. I had forgotten that Alexander went back and revised the monte carlo labs. These issues apply primarily to the version we had when I was the TA. The biggest issue was that the problems were nebulous and hard to follow. At a glance, that appears to be better now.
